tetrateci/create_istio_release.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd ..
+git clone https://github.com/istio/release-builder --depth=1
+envsubst < ./istio/tetrateci/manifest.yaml.in > ./release-builder/manifest.yaml
+cd release-builder
+cp -r ../istio .
+export IMAGE_VERSION=$(curl https://raw.githubusercontent.com/istio/test-infra/master/prow/config/jobs/release-builder.yaml | head -n 4 | tail -n 1 | cut -d: -f3)
```

Comment:
grep for the image name without the tag and use that info. This would break if somebody adds extra line to the file.